Tests failing on Main

Failing for Mac only
```
Run just test
pytest --cov-report xml --cov-report html --cov=bioio_bioformats bioio_bioformats/tests
============================= test session starts =============================
platform win32 -- Python 3.10.11, pytest-8.4.1, pluggy-1.6.0
rootdir: D:\a\bioio-bioformats\bioio-bioformats
configfile: pyproject.toml
plugins: cov-6.2.1, raises-0.11
collected 27 items

Windows fatal exception: access violation

Current thread 0x00001ac4 (most recent call first):
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\jpype\_core.py", line 357 in startJVM
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\scyjava\_jvm.py", line 191 in start_jvm
  File "D:\a\bioio-bioformats\bioio-bioformats\bioio_bioformats\utils.py", line 665 in _try_get_loci
  File "D:\a\bioio-bioformats\bioio-bioformats\bioio_bioformats\biofile.py", line 87 in __init__
  File "D:\a\bioio-bioformats\bioio-bioformats\bioio_bioformats\reader.py", line 141 in __init__
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\bioio_base\test_utilities.py", line 177 in run_image_file_checks
  File "D:\a\bioio-bioformats\bioio-bioformats\bioio_bioformats\tests\test_reader.py", line 251 in test_bioformats_reader
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\_pytest\python.py", line 157 in pytest_pyfunc_call
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\pluggy\_callers.py", line 121 in _multicall
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\pluggy\_manager.py", line 120 in _hookexec
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\pluggy\_hooks.py", line 512 in __call__
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\_pytest\python.py", line 1671 in runtest
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\_pytest\runner.py", line 178 in pytest_runtest_call
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\pluggy\_callers.py", line 121 in _multicall
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\pluggy\_manager.py", line 120 in _hookexec
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\pluggy\_hooks.py", line 512 in __call__
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\_pytest\runner.py", line 246 in <lambda>
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\_pytest\runner.py", line 344 in from_call
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\_pytest\runner.py", line 245 in call_and_report
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\_pytest\runner.py", line 136 in runtestprotocol
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\_pytest\runner.py", line 117 in pytest_runtest_protocol
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\pluggy\_callers.py", line 121 in _multicall
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\pluggy\_manager.py", line 120 in _hookexec
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\pluggy\_hooks.py", line 512 in __call__
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\_pytest\main.py", line 367 in pytest_runtestloop
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\pluggy\_callers.py", line 121 in _multicall
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\pluggy\_manager.py", line 120 in _hookexec
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\pluggy\_hooks.py", line 512 in __call__
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\_pytest\main.py", line 343 in _main
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\_pytest\main.py", line 289 in wrap_session
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\_pytest\main.py", line 336 in pytest_cmdline_main
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\pluggy\_callers.py", line 121 in _multicall
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\pluggy\_manager.py", line 120 in _hookexec
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\pluggy\_hooks.py", line 512 in __call__
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\_pytest\config\__init__.py", line 175 in main
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\_pytest\config\__init__.py", line 201 in console_main
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\Scripts\pytest.exe\__main__.py", line 10 in <module>
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\runpy.py", line 86 in _run_code
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\runpy.py", line 196 in _run_module_as_main
Windows fatal exception: access violation

Current thread 0x00001ac4 (most recent call first):
  File "D:\a\bioio-bioformats\bioio-bioformats\bioio_bioformats\biofile.py", line 90 in __init__
  File "D:\a\bioio-bioformats\bioio-bioformats\bioio_bioformats\reader.py", line 141 in __init__
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\bioio_base\test_utilities.py", line 177 in run_image_file_checks
  File "D:\a\bioio-bioformats\bioio-bioformats\bioio_bioformats\tests\test_reader.py", line 251 in test_bioformats_reader
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\_pytest\python.py", line 157 in pytest_pyfunc_call
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\pluggy\_callers.py", line 121 in _multicall
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\pluggy\_manager.py", line 120 in _hookexec
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\pluggy\_hooks.py", line 512 in __call__
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\_pytest\python.py", line 1671 in runtest
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\_pytest\runner.py", line 178 in pytest_runtest_call
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\pluggy\_callers.py", line 121 in _multicall
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\pluggy\_manager.py", line 120 in _hookexec
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\pluggy\_hooks.py", line 512 in __call__
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\_pytest\runner.py", line 246 in <lambda>
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\_pytest\runner.py", line 344 in from_call
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\_pytest\runner.py", line 245 in call_and_report
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\_pytest\runner.py", line 136 in runtestprotocol
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\_pytest\runner.py", line 117 in pytest_runtest_protocol
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\pluggy\_callers.py", line 121 in _multicall
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\pluggy\_manager.py", line 120 in _hookexec
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\pluggy\_hooks.py", line 512 in __call__
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\_pytest\main.py", line 367 in pytest_runtestloop
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\pluggy\_callers.py", line 121 in _multicall
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\pluggy\_manager.py", line 120 in _hookexec
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\pluggy\_hooks.py", line 512 in __call__
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\_pytest\main.py", line 343 in _main
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\_pytest\main.py", line 289 in wrap_session
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\_pytest\main.py", line 336 in pytest_cmdline_main
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\pluggy\_callers.py", line 121 in _multicall
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\pluggy\_manager.py", line 120 in _hookexec
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\pluggy\_hooks.py", line 512 in __call__
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\_pytest\config\__init__.py", line 175 in main
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\_pytest\config\__init__.py", line 201 in console_main
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\Scripts\pytest.exe\__main__.py", line 10 in <module>
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\runpy.py", line 86 in _run_code
  File "C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\runpy.py", line 196 in _run_module_as_main
error: Recipe `test` failed on line 39 with exit code 2816
bioio_bioformats\tests\test_reader.py F
Error: Process completed with exit code 1.
```

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -51,6 +51,12 @@ jobs:
       uses: actions/setup-python@v5
       with:
         python-version: ${{ matrix.python-version }}
+    - name: Set up Java (Windows only)
+      if: runner.os == 'Windows'
+      uses: actions/setup-java@v3
+      with:
+        distribution: temurin
+        java-version: 17
     - uses: extractions/setup-just@v3
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}

